core/core.go

```diff
@@ -43,6 +44,7 @@ type Textile struct {
 
 	threadservice s.ServiceBoostrapper
 
+	threadsServiceServer *serviceapi.Server
```

Comment:
Yet another server. I could see separating all these at some point. Thread store adapters could actually use the go client.